openapi/components/schemas/UpcomingInvoice.yaml

```
@@ -34,7 +34,7 @@ properties:
     type: integer
   organizationId:
     allOf:
-      - $ref: ./Organizations/OrganizationId.yaml
+      - $ref: ./OrganizationId.yaml
   items:
     type: array
     description: Upcoming Invoice items array.
```

nitpick: capitalization 
```suggestion
    description: Upcoming invoice items array.
```